test/drivers/pyscfd/test_driver_problem_pyscf.py

```diff
@@ -122,6 +123,7 @@ def test_lih_rhf_bk(self):
         )
         self._assert_energy_and_dipole(result, "lih")
 
+    @slow_test
```

Comment:
I am not sure these are all that slow. I would be reluctant not to have these in there running normally under CI.

Reply:
I do not see the benefit of the `OH` tests compared to the existing `H2` *and* `LiH` tests. The `OH` tests are, relative to the other ones, *very* slow. Hence, I disabled them. If you disagree strongly we can re-enable them. However, for the record, I don't think they are necessary.

Reply:
The doublet OH was included since its not a closed shell molecule to start with unlike LiH. Its a bit bigger and hence the bit slower. It was to make sure we were doing things right with either type of molecule across the different methods - it was originally called test driver methods. How long does it take to run that you consider it slow? If anything I could support perhaps putting some of them oh_uhf/oh_rohf (same for lih_uhf/rohf/rhf) as slow where they test the same thing but with a different mapping to thin things down. Arguably if it works for one mapping it should be good for the others - if it does not its more likely the mapper is broke than the driver. Having said that I am not sure we have mapping tests with such coverage of open shell molecules elsewhere.